wp-github-sync.php

```diff
@@ -253,3 +253,9 @@ public static function write_log($msg, $write = 'line') {
 }
 
 $wpghs = new WordPress_GitHub_Sync;
+
+function get_the_github_url() {
```

Comment:
Personally, probably edit, since presumably you just read the article on the site. In which case, I should probably update the link accordingly.